### DISCUSSION TITLE
Paperless-NGX 403 Forbidden after Account Creation

### DISCUSSION THREAD

**Discussion by AinzDerErste** [tteck/Proxmox#3854] [Help Needed] [answered]
Hello, I can only Log in to Paperless with the IP. When I want to use my Domain, I am getting a 403 Forbidden Error (CSRF-Verification failed).

I added the PAPERLESS_URL in the paperless.conf and restarted the LXC Container but it still does not work.

**Comment by Mellowlynx** [comment, marked as answer]
Make sure that `PAPERLESS_URL=` is set with **https://**, not just the (sub-)domain.
You could try to also set `PAPERLESS_CSRF_TRUSTED_ORIGINS=` but this one should not be necessary.

If you use a reverse proxy, [read this](https://github.com/paperless-ngx/paperless-ngx/issues/817#issuecomment-1115306658), there are some tips that worked for others.
